app/components/pricing-page-header/template.hbs

```diff
@@ -0,0 +1,6 @@
+{{#if isDevelopment}}
+  <h1>Add a Payment Method to Your<br /> Pay-as-you-go Development Plan</h1>
```

Comment:
Tiny, unimportant nit:

In HTML5, you are never required to use a `/>`. Instead, `hr` tags are part of a group of tags called `void`, meaning they can have no children element. `img` is another example.

Death to `/>`!